esp/src/src-react/components/Title.tsx

```diff
@@ -27,6 +27,35 @@ export const DevTitle: React.FunctionComponent<DevTitleProps> = ({
         return retVal;
     })];
 
+    const advMenuProps: IContextualMenuProps = {
+        items: [
+            { key: "errors", href: "#/errors", text: "Error/Warning(s)", },
+            { key: "divider_1", itemType: ContextualMenuItemType.Divider },
+            { key: "banner", text: "Set Banner" },
+            { key: "toolbar", text: "Set Toolbar" },
+            { key: "divider_2", itemType: ContextualMenuItemType.Divider },
+            { key: "docs", href: "https://hpccsystems.com/training/documentation/", text: "Documentation", target: "_blank" },
+            { key: "downloads", href: "https://hpccsystems.com/download", text: "Downloads", target: "_blank" },
+            { key: "releaseNotes", href: "https://hpccsystems.com/download/release-notes", text: "Release Notes", target: "_blank" },
+            { key: "additionalResources", text: "Additional Resources", subMenuProps: {
+                items: [
+                    { key: "redBook", href: "https://wiki.hpccsystems.com/display/hpcc/HPCC+Systems+Red+Book", text: "Red Book", target: "_blank" },
+                    { key: "forums", href: "https://hpccsystems.com/bb/", text: "Forums", target: "_blank" },
+                    { key: "issues", href: "https://track.hpccsystems.com/issues/", text: "Issue Reporting", target: "_blank" },
+                ]
+            }},
+            { key: "divider_3", itemType: ContextualMenuItemType.Divider },
+            { key: "lock", text: "Lock" },
+            { key: "logout", text: "Logout" },
+            { key: "divider_4", itemType: ContextualMenuItemType.Divider },
+            { key: "config", text: "Configuration" },
+            { key: "about", text: "About" },
```

Comment:
All the "text" elements should have a corresponding item in "nlsHPCC" (for auto translations).